<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" hash="2486493475">
    <data-source source="LOCAL" name="Django default" uuid="a8181fd1-9247-49f3-9cf7-c719fbaa8d7b">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/CS361DjangoTest/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\amernoureddin\PycharmProjects\CS361DjangoTest\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/xerial-sqlite-license.txt</url>
        </library>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/sqlite-jdbc-3.8.9.jar</url>
        </library>
      </libraries>
    </data-source>
    <data-source source="LOCAL" name="Django default2" uuid="4d71d9f6-eaa8-4f3a-a0a4-571569364951">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/CS361DjangoTest/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\amernoureddin\PycharmProjects\CS361DjangoTest\db.sqlite3</jdbc-url>
      <explain-plan-config>
        <provider>com.intellij.database.plan.ExplainPlanProvider.NullExplainPlanProvider</provider>
      </explain-plan-config>
    </data-source>
  </component>
</project>